project/.github/ISSUE_TEMPLATE.md

```diff
@@ -12,21 +12,20 @@
 
 #### Sonata packages
 
-```bash
-$ composer show sonata-project/*
-# Put the result here.
+```
+$ composer outdated --all 'sonata-project/*'
+# Put the result here if there is one, otherwise, just remove this paragraph.
 ```
 
 #### Symfony packages
 
-```bash
-$ composer show symfony/*
-# Put the result here.
+```
+$ composer outdated --all 'symfony/*'
 ```
 
 #### PHP version
 
-```bash
```

Comment:
No it should really be removed. Don't forget, in the end, we will get the output of the composer, which is certainly not a bash script inside this fenced block.